src/main/java/tools/jackson/dataformat/xml/JacksonXmlAnnotationIntrospector.java

```diff
@@ -208,6 +209,10 @@ public PropertyName findNameForDeserialization(MapperConfig<?> config, Annotated
         PropertyName pn = PropertyName.merge(_findXmlName(a),
                 super.findNameForDeserialization(config, a));
         if (pn == null) {
+            if(_hasAnnotation(a, JacksonXmlText.class)){
+                return PropertyName.construct(FromXmlParser.DEFAULT_TEXT_PROPERTY);
```

Comment:
Quick notes: not just `_hasAnnotation` -- `JacksonXmlText.value()` must be `true` (specifying `false` can be used to disable annotation's effect).

But I have related concern: there now won't be a way to override name of logical property to use as it's effectively hard-coded. Should there be a way to re-configure that? And if so, how?
(`MapperConfig` passed won't have access to XML-specific config mapper has. But maybe this introspector should have its own config or something.

Reply:
For the first note: This has been fixed in the latest commit of this pull request

For the second note: I've added a config on the introspector so that this can be configured.